README.md

```diff
@@ -822,4 +822,4 @@ it is not clear that rollback will result in a working configuration.
 
 
 *Want to contribute? Take a look at our [contributing
-guidelines](https://github.com/linux-system-roles/network/blob/master/contributing.md)!*
```

Comment:
Please group this together with other changes about the branch name. Thanks.

Reply:
Moved to another PR.